<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormExpert</class>
 <widget class="QWidget" name="FormExpert">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>977</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="hLayFunc">
     <item>
      <widget class="QToolButton" name="btnSerial">
       <property name="text">
        <string>Serial</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnData">
       <property name="text">
        <string>Data</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnPlot">
       <property name="text">
        <string>Plot</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="tBtnCorrection">
       <property name="text">
        <string>Correction</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tBtnSimulate">
       <property name="text">
        <string>Simulate</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tBtnHistory">
       <property name="text">
        <string>History</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tBtnData">
       <property name="text">
        <string>Data</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="btnUpdate">
       <property name="text">
        <string>update</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnExternal">
       <property name="text">
        <string>external</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnLog">
       <property name="text">
        <string>Log</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnSetting">
       <property name="text">
        <string>Setting</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QStackedWidget" name="stackedWidget"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
